tools/Dashboard/DevHome.Dashboard/ViewModels/WidgetViewModel.cs

```diff
@@ -24,8 +25,8 @@
 using Microsoft.UI.Xaml.Controls;
 using Microsoft.Windows.Widgets;
 using Microsoft.Windows.Widgets.Hosts;
+using Newtonsoft.Json.Linq;

```

Comment:
To encourage not overusing newtonsoft, can we remove the using and explicitly qualify where we need to down in the code?